### PR TITLE
Unblock subsequent Singleton.createInstance()

### PR DESCRIPTION
**Pull request by Nokel81**
- Also fix clusters with unknown workspaces

Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

**Comment by aleksfront** [comment]
In the part of ignoring clusters without `workspace` attribute it works as expected 👍🏻 

However, can you explain a bit more why this singleton fixes is needed and what cases this PR fixes?

**Comment by Nokel81** [comment]
The singleton changes are needed because if the constructor of a singleton fails, and even if something else catches that errors, subsequent attempts to make new singletons (even of different types) will all fail.

File: build/set_build_version.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ function getBuildChannel(): string {
     case "beta":
       return "beta";
     case undefined:
+    case "latest":
```
Comment (jakolehm) [inline build/set_build_version.ts]: Unrelated change?

**Review by jakolehm — APPROVED** [review]
LGTM
